modules/govuk/manifests/apps/email_alert_api/sidekiq_queue_check.pp

```diff
@@ -0,0 +1,37 @@
+# == Define: govuk::apps::email_alert_api::sidekiq_queue_checks
+#
+# Creates an Icinga check which checks Graphite for activity
+# on Sidekiq queues
+#
+# === Parameters
+#
+# [*ensure*]
+#   Whether to enable the check.
+#   Default: present
+#
+# [*title*]
+#   The name of the sidekiq queue to check.
+#
+# [*size_warning*]
+#   The number of messages on a queue to trigger a warning.
+#
+# [*size_critical*]
+#   The number of messages on a queue to trigger a critical alert.
+#
+define govuk::apps::email_alert_api::sidekiq_queue_check(
+  $size_warning,
```

Comment:
I wonder if we should add defaults to these because I imagine for most of the queues these numbers will be the same, but that's something for the next PR I think.